production/blog/mythoughtsonfood.html

```diff
@@ -22,8 +22,8 @@
 	<link rel="stylesheet" href="/resources/css/reset.css">
```

Comment:
Don't bother renaming now, but in the future name files like so for SEO optimization: 
`my-thoughts-on-food.html`

These files have already been indexed so don't bother.